### ISSUE TITLE
Quick way to publish site

### ISSUE THREAD

**Issue by suzumiyasmith** [HelloZeroNet/ZeroNet#879]
I am developing a multi-user site.
While debugging, I find it's painful to test.

Now, I develop my site on a local machine and test it on a vps.
I write a watch script: Every time there is an update in my site data folder, the script will run ` zeronet.py siteSign myAddress myPrivateKey` and `zeronet.py sitePublish myAddress myVpsIP 15441`. But the problem is that there is a huge latency for sitePublish: When I check if it is updated in remote vps via local browser, I find I have to wait at least 5 min to see updated website.

For details, while publishing, it says `Thanks, file content.json updated! 1/5` in almost no time. But when I check it via browser for the status of the site on vps, I find that it shows `updating 2 left`. I don't know why. And sitePublish could only run for 60s. After that, the site on vps says that `2 file update fail`.

If I download normal files from my vps, it could be more than 1M/s. I am wondering why only `content.js` could be transferred in a few seconds leaving other files update fail.

**Comment by ysc3839** [comment]
You can open the side menu, turn on "This is my site", and then click "Sign" and "Publish".

**Comment by HelloZeroNet** [comment]
You can check the failed files on the sidebar. JS/CSS files are cached by the browser for 5 minutes, so for development: open Developers Tools (F12) then check Disable Cache on Network tab. (Chrome)

**Comment by suzumiyasmith** [comment]
I know there are missing files listed in the slide bar. As for cache, I think that reloading tab with override cache should not be a problem. 

The main problem I faced is that after I update and sign site, the site could only be served for max 60 secs and in that duration, the files I updated could not be reached by my remote VPS which IP and port are provided in the command.

In other words, only `content.json` has been updated, but the files listed haven't been updated.
